test/Microsoft.ML.Tests/ImagesTests.cs

```diff
@@ -201,13 +201,13 @@ public void TestBackAndForthConversionWithAlphaInterleave()
             var cropped = new ImageResizingTransformer(env, "ImageCropped", imageWidth, imageHeight, "ImageReal").Transform(images);
 
             var pixels = new ImagePixelExtractingTransformer(env, "ImagePixels", "ImageCropped", ImagePixelExtractingEstimator.ColorBits.All, true, 2f / 255, 127.5f).Transform(cropped);
-            IDataView backToBitmaps = new VectorToImageTransform(env, new VectorToImageTransform.Arguments()
+            IDataView backToBitmaps = VectorToImageConvertingTransformer.Create(env, new VectorToImageConvertingTransformer.Options()
```

Comment:
>VectorToImageConvertingTransformer.Create [](start = 38, length = 41)

Please don't do that.

First it's a different style comparing to all other transformers.

Second Create method is IDataTransform which we want to get rid of.